routers/api/v1/repo/runners.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package repo
+
+import (
+	"errors"
+	"net/http"
+
+	actions_model "code.gitea.io/gitea/models/actions"
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/util"
+	"code.gitea.io/gitea/services/convert"
+)
+
+// GenerateRunnerToken generate a new runner token for a repository.
+func GenerateRunnerToken(ctx *context.APIContext) {
```

Comment:
Are there any checks to see if Actions are enabled for the instance/org/repo?

Reply:
True, we should probably check that.